openday_scavenger/api/puzzles/service.py

```diff
@@ -193,6 +193,7 @@ def create(db_session: Session, puzzle_in: PuzzleCreate) -> Puzzle:
     # explicitly. This maintains a nice abstraction between the service layer
     # and the database layer.
     puzzle = Puzzle(
+        id=puzzle_in.id,
```

Comment:
Same change as https://github.com/AustralianSynchrotron/openday-scavenger/pull/142
Required so that the tests can be run multiple times without crashing on a fresh DB